mikrotik/provider.go

```diff
@@ -2,32 +2,50 @@ package mikrotik
 
 import (
 	"context"
+	"fmt"
+	"strings"
 
 	mt "github.com/ddelnano/terraform-provider-mikrotik/client"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+func init() {
+	schema.SchemaDescriptionBuilder = func(s *schema.Schema) string {
+		desc := s.Description
+		// add default value in description, if it was declared in the resource's schema.
+		if s.Default != nil {
+			if s.Default == "" {
+				desc += " Default: `\"\"`."
+			} else {
+				desc += fmt.Sprintf(" Default: `%v`.", s.Default)
+			}
+		}
+
+		return strings.TrimSpace(desc)
+	}
```

Comment:
It looks like this is used in the other PRs (#84, #85, #86, etc). Can we make this a function that each of the resources can share?

```go
func buildSchemaDescriptionBuilder(s *schema.Schema) string {
		desc := s.Description
		// add default value in description, if it was declared in the resource's schema.
		if s.Default != nil {
			if s.Default == "" {
				desc += " Default: `\"\"`."
			} else {
				desc += fmt.Sprintf(" Default: `%v`.", s.Default)
			}
		}

		return strings.TrimSpace(desc)
	}

func init() {
	schema.SchemaDescriptionBuilder = buildSchemaDescriptionBuilder
}
```